backstop/tests/table.html

```diff
@@ -273,15 +273,66 @@ <h2>Default</h2>
             </div>
             <div class="iui-cell iui-slot">
               <button
-                class="iui-button iui-borderless"
+                class="iui-button iui-borderless iui-dropdown"
                 aria-label="Modify columns"
                 type="button"
+                onclick="openColumnManger()"
               >
                 <svg-column-manager
                   class="iui-button-icon"
                   aria-hidden="true"
                 ></svg-column-manager>
               </button>
+              <div
+                data-tippy-root=""
+                id="tippy-1"
+                style="z-index: 9999; display: none; position: fixed; inset: 0px auto auto 0px; margin: 0px; transform: translate(625px, 62px);"
+              >
+                <div
+                  class="tippy-box iui-popover"
+                  data-state="hidden"
+                  data-placement="bottom-end"
+                  style="transition-duration: 0ms;"
+                >
+                  <div
+                    class="tippy-content"
+                    data-state="hidden"
+                    style="transition-duration: 0ms;"
+                  >
+                    <div>
+                      <ul
+                        class="iui-menu"
+                        role="menu"
+                        style="min-width: 58px;"
+                      ><label
+                          class="iui-checkbox-wrapper iui-menu-item"
+                          style="cursor: pointer;"
+                        ><input
```

Comment:
Why are you not using our `iui-checkbox` component?

Reply:
Yeah, you can clean whole html up by removing hardcoded inline styles.. I thinnk you copied this whole thing from wrong place. Check our checkbox html